practices/host-crucial-conversation.md

```diff
@@ -0,0 +1,70 @@
+# Host a Crucial Conversation
+
+By hosting a crucial conversation, you're helping team members with differing (sometimes strong) opinions navigate high-stakes and sensitive discussions. You're also helping to strengthen relationships and build trust within your team. 
```

Comment:
After these first two sentences, I suggest adding an example of what a crucial conversation might look like. What might be the topic(s) discussed? e.g., "For a team of software developers, a crucial conversation might be centered around [xyz or xyz]."